content/docs/atom-guide/look-dev/materials/wrinkle-masks.md

```diff
@@ -0,0 +1,60 @@
+
+{{< important >}}
+This workflow represents a minimum viable product for animating wrinkle masks. There are a number of aspects that could use feedback on how to improve, such as how the normal maps and blend masks are blended, and the potential need for corrective blend masks. If you are using this feature, please reach out to [sig-graphics-audio on Discord](https://discord.com/channels/805939474655346758/816043793576886273) with your suggestions!
+{{< /important >}}
+
+## Folder Structure and Naming Conventions
+In the folder that `<mycharacter>.fbx` is in, add a `<mycharacter>_wrinklemasks` folder. Using this exact naming convention, the Asset Processor will associate any wrinkle masks in the wrinkle mask folder with morph targets from the .fbx.
+
+![Folder Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-folder-naming-convention.png)
+
+In the `<mycharacter>_wrinklemasks` folder add one blend mask for each morph target that activates the wrinkle layers. If the morph target does not activate any wrinkle layers at all, it does not need a mask. The name of each mask must match the name of the morph target/blend shape/shape key in the fbx. If you rename the blend shape in the fbx, you must rename the wrinkle mask. Each mask is an independent file, and there is no limitation on how many wrinkle masks you can add to a character, although there will be at most 16 active at any one time.
+
+![Wrinkle Mask File Naming Convention](/images/atom-guide/materials/skin/wrinkle-mask-file-naming-convention.png)
+
+Here you can see that the wrinkle mask file names above match the morph target (shape key in Blender) names below, ending with _wrinklemask.tif.
+
+![Shape Key Names in Blender](/images/atom-guide/materials/skin/shape-key-names-in-blender.png)
+
+At present, the masks cannot be shared, so even if you have two morph targets that would use the same mask, you'll have to duplicate the mask texture. The morph targets do not all have to have masks. The morph targets will still function without a mask, they just won't drive the wrinkle layers.
+
+Each mask should be authored as a RGBA .tif file. The skin shader supports up to 4 wrinkle layers (normal and/or basecolor). Each color channel corresponds to the weight of one of the wrinkle layers. So for a material with only one wrinkle layer, all of the masks will be red-only (though the textures themselves will still be RGBA, with all black in the green, blue, and alpha channels). If a material uses two wrinkle layers (for example, one for vertical wrinkles and one for horizontal wrinkles), then each wrinkle mask will be red, green, or a combination of the two.
```

Comment:
I'm not sure if this paragraph fits in this section. Parts of this fifth paragraph sounds like it belongs in Wrinkle Mask File Properties. The other parts are better suited in a Usage section, or similar. I can provide more help with this.